### PR TITLE
Add area mapping and test data

### PR DESCRIPTION
**Pull request by vimto**

### CODE REVIEW

File: packages/IngestAPI/src/main/kotlin/nz/govt/eop/Messages.kt [hunk 1]
```diff
@@ -1,11 +1,12 @@
 package nz.govt.eop.messages
 
+import java.math.BigDecimal
 import java.time.Instant
 import nz.govt.eop.ingestapi.WaterAllocation
 
 data class WaterAllocationMessage(
     val areaId: String,
-    val amount: Int,
+    val amount: BigDecimal,
```
Comment (vimto) [inline packages/IngestAPI/src/main/kotlin/nz/govt/eop/Messages.kt:9]: This was based on Jooqs default mapping for Postgres numeric types, and matches what we use for `water_allocations.allocation_amount`

File: packages/Manager/src/main/resources/db/migration/V0019__change_water_alloction_amount_to_numeric.sql [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+UPDATE allocation_amounts SET area_id = 'RaumatiGW' WHERE id = 41;
+UPDATE allocation_amounts SET area_id = 'TaratahiGW' WHERE id = 5;
+UPDATE allocation_amounts SET area_id = 'TauherenikauGW' WHERE id = 9;
+UPDATE allocation_amounts SET area_id = 'Te HoroGW' WHERE id = 43;
+UPDATE allocation_amounts SET area_id = 'Te Ore OreGW' WHERE id = 1;
+UPDATE allocation_amounts SET area_id = 'Upper HuttGW' WHERE id = 35;
+UPDATE allocation_amounts SET area_id = 'Upper RuamahangaGW' WHERE id = 3;
+UPDATE allocation_amounts SET area_id = 'WaikanaeGW' WHERE id = 42;
+UPDATE allocation_amounts SET area_id = 'WaingawaGW' WHERE id = 2;
+UPDATE allocation_amounts SET area_id = 'WaitohuGW' WHERE id = 44;
```
Comment (vimto) [inline packages/Manager/src/main/resources/db/migration/V0019__change_water_alloction_amount_to_numeric.sql:44]: @MartyP233 do you want to quickly spot-check this import based on the spreadsheet you've created?

File: packages/IngestAPI/src/main/kotlin/nz/govt/eop/Messages.kt (hunk 1 above)
Comment (smozely) [reply packages/IngestAPI/src/main/kotlin/nz/govt/eop/Messages.kt:9]: 👍 sweet, yeah avoid all the IEEE floating point pain

**Review by smozely — APPROVED** [review]
Sweet to me

**Review by MartyP233 — APPROVED** [review]
checked against the spreadsheet - names and id's match up.

**Pull request merged**